### DISCUSSION TITLE
How can I uninstall NextFlow?

### DISCUSSION THREAD

**Discussion by marco-brandizi** [nextflow-io/nextflow#2256] [Q&A] [answered]
Hi all, 

apologies if the question is too trivial but I'm new to NextFlow and I always like to know how I can remove a software product before installing it. I cannot find a documented way for NextFlow, I understand everything goes into `$HOME/.nextflow/` and the `nextflow` script, is that correct?

Thanks in advance,
Marco.

**Comment by abhi18av** [comment, marked as answer]
Hi @marco-brandizi ,

> I understand everything goes into $HOME/.nextflow/ and the nextflow script

Yes that is correct (as far as I understand) 👍
